.env-docker

```diff
@@ -0,0 +1,4 @@
+MYSQL_ROOT_PASSWORD=mypass
+SECRET_KEY_BASE=abcdefg
+VIRTUAL_HOST=beer.docker
```

Comment:
Another way is to separate the env file that needed to separated environments to the other container, I'd just noticed the declared environment variables are redundant to others so I'd use the env file to remove the redundancy and make it more flexible if adding another environment variable.